### ISSUE TITLE
[Suggestion] Romantic date

### ISSUE THREAD

**Issue by RavenPLAYER** [Monika-After-Story/MonikaModDev#5004]
And what if we could have a romantic "date" with Monika? Close the curtains and put a candle in the center of the table, relaxing music and enjoy some time with it. A completely dark room and that the candlelight gives an incredible touch, although it is preferable that it can only be done at night. Thanks for reading!

**Comment by FrikiHD** [comment]
Oh yeah, that sure is a good idea

**Comment by lunulae** [comment]
I like this idea, though it might be tricky to do a candle on the desk because of a need for a new light filter on the sprites.
I really like the idea of having a repeatable event like this though, something you plan for ahead of time and have to meet her for. Once the framework for giving her food and drinks is finished it could even be possible to plan for a lunch or dinner together.

**Comment by ThePotatoGuy** [comment]
duplicate of #763
[closing as duplicate]
